pkg/cli/upgradeassistant/cmd/migrate/1110.go

```diff
@@ -0,0 +1,115 @@
+package migrate
+
+import (
+	"context"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/koderover/zadig/pkg/cli/upgradeassistant/internal/repository/models"
+	"github.com/koderover/zadig/pkg/cli/upgradeassistant/internal/upgradepath"
+	"github.com/koderover/zadig/pkg/tool/log"
+)
+
+func init() {
+	upgradepath.RegisterHandler("1.10.0", "1.11.0", V1100ToV1110)
+	upgradepath.RegisterHandler("1.11.0", "1.10.0", V1110ToV1100)
+}
+
+func V1100ToV1110() error {
+	// iterate all roles ， if the role contains manage env , add debug_pod
+	if err := roleAddPodDebug(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func V1110ToV1100() error {
+	if err := roleDeletePodDebug(); err != nil {
+		return err
+	}
+	return nil
+}
+
+func roleAddPodDebug() error {
+	log.Info("start to scan role")
+	ctx := context.Background()
+	var res []*models.Role
+	cursor, err := newRoleColl().Find(ctx, bson.M{})
+	if err != nil {
+		log.Errorf("Failed to Find Policies, err: %s", err)
+		return err
+	}
+	err = cursor.All(ctx, &res)
+	if err != nil {
+		log.Errorf("get roles err:%s", err)
+		return err
+	}
+	for _, role := range res {
+		log.Infof("start to check role:%s\n", role.Name)
```

Comment:
Why does adding a \n will not automatically wrap the line?

Reply:
removed